src/main/java/com/mixpanel/android/util/HttpService.java

```diff
@@ -48,7 +48,7 @@ public boolean isOnline(Context context) {
     }
 
     @Override
-    public byte[] performRequest(String endpointUrl, List<NameValuePair> params, SSLSocketFactory socketFactory) throws ServiceUnavailableException, IOException {
+    public byte[] performRequest(String endpointUrl, Map<String, Object> params, SSLSocketFactory socketFactory) throws ServiceUnavailableException, IOException {
```

Comment:
NameValuePair has been removed in API 23. Basically, most of the stuff in `org.apache.http` has been removed in 23.